### PR TITLE
Update Fanli.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
https://m.fanli.com
https://quan.fanli.com
http://static2.51fanli.net/common/libs/headjs/head.min.js

最近都懒得自己提PR了……

File: src/chrome/content/rules/51fanli.net.xml [hunk 1]
```diff
@@ -1,45 +1,27 @@
...
-
 -->
-<ruleset name="51 Fanli.net (partial)">
 
-	<!--	Direct rewrites:
-				-->
+<ruleset name="51fanli.net">
+
 	<target host="51fanli.net" />
 	<target host="imagefile2.51fanli.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/51fanli.net.xml]: imagefile.51fanli.net

File: src/chrome/content/rules/51fanli.net.xml [hunk 2]
```diff
@@ -1,45 +1,27 @@
...
 	<target host="l3.51fanli.net" />
 	<target host="l4.51fanli.net" />
-
+	<target host="spic0.51fanli.net" />
+	<target host="spic1.51fanli.net" />
+	<target host="spic2.51fanli.net" />
+	<target host="spic3.51fanli.net" />
+	<target host="spic4.51fanli.net" />
+	<target host="spic5.51fanli.net" />
+	<target host="static2.51fanli.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/51fanli.net.xml]: static.51fanli.net

File: src/chrome/content/rules/Fanli.com.xml [hunk 3]
```diff
@@ -1,69 +1,46 @@
...
+	<target host="ubt1.fanli.com" />
+	<target host="ubt2.fanli.com" />
+	<target host="ubt3.fanli.com" />
+	<target host="ubt4.fanli.com" />
+	<target host="ubt5.fanli.com" />
+	<target host="ubt6.fanli.com" />
+	<target host="ubt7.fanli.com" />
+	<target host="ubt8.fanli.com" />
+	<target host="ubt9.fanli.com" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/Fanli.com.xml]: See also:
haitao.fanli.com
travel.fanli.com

File: src/chrome/content/rules/Fanli.com.xml [hunk 4]
```diff
@@ -13,8 +13,12 @@
 		- passport
 		- super
 		- taobao
+		- travel
 		- xiaozu
 		- zhide
+
+	Mismatch:
+		imagefile.51fanli.com	https://imagefile.51fanli.com/UploadPic/2010/8/17/2010817192827470.jpg
```
Comment (J0WI) [inline src/chrome/content/rules/Fanli.com.xml]: You should investigate in `51fanli.net.xml`

**Pull request merged**